servers/src/main/java/tachyon/worker/block/meta/StorageDir.java

```diff
@@ -465,7 +448,28 @@ public void cleanupUserTempBlocks(long userId, List<Long> tempBlockIds) {
     return userTempBlocks;
   }
 
+  /**
+   * @return the block store location of this directory.
+   */
   public BlockStoreLocation toBlockStoreLocation() {
     return new BlockStoreLocation(mTier.getTierAlias(), mTier.getTierLevel(), mDirIndex);
   }
+
+  private void reserveSpace(long size, boolean committed) {
+    Preconditions.checkState(size <= mAvailableBytes.get(),
+        "Available bytes should always be non-negative");
+    mAvailableBytes.addAndGet(-size);
+    if (committed) {
+      mCommittedBytes.addAndGet(size);
+    }
+  }
+
+  private void reclaimSpace(long size, boolean committed) {
```

Comment:
yes, but I feel we don't strongly enforce alphabetically ordering of methods. Shall we be more strict on this?